package.json

```diff
@@ -7,7 +7,8 @@
     "start": "node src/app.js",
     "test": "node tests/server_test.js",
```

Comment:
Fix your test script to run all the tests

Reply:
@superhealthyminds/developers can we confirm that this has been completed? Or is there a plan to complete it at a later date?